test/expected/chunk_utils.out

```diff
@@ -771,13 +766,65 @@ ORDER BY c.id;
        12 |             2 | _timescaledb_internal | _hyper_2_12_chunk |           6 |         7
 (6 rows)
 
+-- We now support show/drop chunks using timestamps/interval even with integer partitioning
+-- the chunk creation time gets used for these. But we need to use "created_before, created_after"
+-- for these
+\set ON_ERROR_STOP 0
+SELECT show_chunks('drop_chunk_test3', older_than => now());
+ERROR:  invalid time argument type "timestamp with time zone"
+SELECT show_chunks('drop_chunk_test2', older_than => now());
+ERROR:  invalid time argument type "timestamp with time zone"
+SELECT show_chunks('drop_chunk_test1', newer_than => INTERVAL '15 minutes');
+ERROR:  invalid time argument type "interval"
+SELECT show_chunks('drop_chunk_test1', older_than => now(), newer_than => INTERVAL '15 minutes');
+ERROR:  invalid time argument type "timestamp with time zone"
+SELECT drop_chunks('drop_chunk_test1', older_than => now());
+ERROR:  invalid time argument type "timestamp with time zone"
+-- mix of older_than/newer_than and created_after/created_before doesn't work
+SELECT show_chunks('drop_chunk_test1', older_than => now(), created_after => INTERVAL '15 minutes');
+ERROR:  invalid time argument type "timestamp with time zone"
+SELECT show_chunks('drop_chunk_test1', created_before => now(), newer_than => INTERVAL '15 minutes');
+ERROR:  invalid time argument type "interval"
+\set ON_ERROR_STOP 1
+SELECT show_chunks('drop_chunk_test3', created_before => now() + INTERVAL '1 hour');
```

Comment:
Using `now()` in tests is risky. I realize you are adding a big interval to be on the safe side, but now() has often led to flaky tests. It is easy to miss something.

I would recommend using a constant or a mock now() function which always return a "safe" timestamp. (I think we've used such mock functions in other tests).

Reply:
We can use `timescaledb.current_timestamp_mock` but this test will unnecessary become a DEBUG only test case because of that.